views/includes/page-templates/relevant-period-statements-add-radios.html

```diff
@@ -0,0 +1,34 @@
+<h4 class="govuk-heading-m">For the pre-registration period</h4>
+<div class="govuk-inset-text">
+    <p>Based on the pre-registration statement you provided earlier in this service, you also need to tell us about any registrable beneficial owners from this period, that the entity has not already told us about.</p>
+    {% include "includes/page-templates/relevant-period.html" %}
+</div>
+
+{{ govukRadios({
+    errorMessage: errors.update.relevant.period.statements if errors,
+    idPrefix: "relevant_period_statements",
+    name: "relevant_period_statements",
```

Comment:
This name field is what determines which radio buttons are in a single group, they are selected exclusive of each other. As mentioned yesterday all the buttons on this page need to be in one group.

Can you have a look in beneficial-owner-type-add-radios.html and change the name here to match then test the behaviour of the radio buttons on the page :)